test/edu/csus/ecs/pc2/core/report/ContestCompareModelTest.java

```diff
@@ -182,46 +180,6 @@ public void testJSONComparisonMatch() throws Exception {
 
     }
 
-    /**
-     * Test compare of pretest CDP and existing event feed file.
-     * 
-     * @throws Exception
-     */
-    public void testReport() throws Exception {
```

Comment:
Why is this file being changed in this PR?  The file doesn't appear to have anything to do with the purpose of the PR...